Masquerade4997Library/src/main/java/Library4997/MasqControlSystems/MasqPurePursuit/MasqPositionTracker.java

```diff
@@ -24,8 +26,7 @@
     private Orientation angles;
     private double zeroPos = 0;
 
-    public MasqPositionTracker(MasqMotor xSystem, MasqMotor ySystem, BNO055IMU imu) {
-        this.imu = imu;
+    public MasqPositionTracker(MasqMotor xSystem, MasqMotor ySystem) {
```

Comment:
Ok, I see what changed and why this isn't in the MasqIMU Constructor. This constructor should call initalizeIMU and the initalizeIMU method should allow you to pass in the name of the imu. So the constructor should take a string 'String imuName' and this should be passed into the initalizeIMU method and then you should use that name.

Reply:
I will make a separate constructor but keep this one because usually the name will always be “imu”

Reply:
ok